### DISCUSSION TITLE
Unable to vertically scroll on very long page

### DISCUSSION THREAD

**Discussion by vaishawil** [gilbarbara/react-joyride#640] [Q&A]
Hi there, i was wondering if it's not possible to vertically auto scroll in v2.3.0? 

I had a case where i got this one long page, and coincidentally the last step was located on the very bottom position of the page, after checking the props docs, i tried to use `disableScrolling`, i also had set the value into `{false}` but it's not working at all. 

The weird thing is i also use `scrollToFirstStep` in case the user need to go back up, and it actually working perfectly well. so i wonder if there is any problem with the `disableScrolling` that makes me cannot auto scroll into the next step? thank you.

**Comment by gilbarbara** [comment]
hey @vaishawil 

Without seeing the code and the app running is impossible to know. Please post a minimal reproduction in codesandbox.

Does the scrolling work in the demo? If it does, it's probably something with your app.

Check the `document.body.clientHeight` and `document.querySelector('your-step-selector').offsetTop`.
The first one should be larger.
